3-async-await/solution.js

```diff
@@ -20,16 +20,59 @@ Example:
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
 
+const { log } = require("console");
+const prices = require("./prices");
+const products = require("./products");
+
 function solution() {
```

Comment:
You can simplify your solution if you add the `async` keyword before the function. Accordingly to point 8, you're allowed to do so.